fe2-android/app/src/test/java/com/github/dedis/popstellar/model/network/method/message/data/rollcall/CloseRollCallTest.java

```diff
@@ -18,7 +18,7 @@
   private final String name = "name";
   private final long time = Instant.now().getEpochSecond();
   private final String location = "Location";
-  private final CreateRollCall createRollCall = new CreateRollCall(name, time, time, location, null, laoId);
+  private final CreateRollCall createRollCall = new CreateRollCall(name, time, time, time, location, null, laoId);
```

Comment:
same naive question with 'time, time, time'. As a first improvement, maybe naming the variable 'now' is closer to the intention of its usage ?